README.md

```diff
@@ -302,6 +305,88 @@ There are some settings that can be configured via a `settings.json` file:
 - VS Code: These settings will show up in the settings window
 - LanguageClient-neovim: Create this file in `$projectdir/.vim/settings.json` or set `g:LanguageClient_settingsPath`
 
+## [hie-bios](https://github.com/mpickering/hie-bios)
+
+`hie-bios` is the way which
+[`hie`](https://github.com/haskell/haskell-ide-engine) sets up a GHC API session.
+
+Given a Haskell project that is managed by Stack, Cabal, or other package tools,
```

Comment:
This information is a bit indirect. Perhaps the documentation here should be more focused on how a few examples of how to configure the project.

Reply:
Tried to be more direct now!